core/src/main/java/org/jobrunr/server/runner/BackgroundJobWithoutIocRunner.java

```diff
@@ -4,15 +4,14 @@
 import org.jobrunr.jobs.JobDetails;
 
 import static org.jobrunr.utils.JobUtils.assertJobExists;
-import static org.jobrunr.utils.reflection.ReflectionUtils.hasDefaultNoArgConstructor;
 
 public class BackgroundJobWithoutIocRunner extends AbstractBackgroundJobRunner {
 
     @Override
     public boolean supports(Job job) {
         JobDetails jobDetails = job.getJobDetails();
         assertJobExists(jobDetails);
-        return !jobDetails.getStaticFieldName().isPresent() && hasDefaultNoArgConstructor(jobDetails.getClassName());
+        return !jobDetails.getStaticFieldName().isPresent();
```

Comment:
What's the reason to remove the `noArgConstuctor` requirement?

Reply:
Kotlin lambda bound variables are passed into the generated class by using constructor parameters